smithy-model/src/main/java/software/amazon/smithy/model/loader/IdlModelParser.java

```diff
@@ -443,11 +444,11 @@ private void parseSimpleShape(ShapeId id, SourceLocation location, AbstractShape
     private void parseCollection(ShapeId id, SourceLocation location, CollectionShape.Builder builder) {
         ws();
         builder.id(id).source(location);
-        parseMembers(id, SetUtils.of("member"));
+        parseMembers(id, SetUtils.of("member"), false);
         modelFile.onShape(builder.id(id));
     }
 
-    private void parseMembers(ShapeId id, Set<String> requiredMembers) {
+    private void parseMembers(ShapeId id, Set<String> requiredMembers, boolean supportsRequiredSugar) {
```

Comment:
Any reason for the naming discrepancy in `supportsRequiredSugar` and `structureMember`?